app/api/views/briefs.py

```diff
@@ -69,10 +72,22 @@ def domain(email):
             errorMessage="Supplier does not have Digital Marketplace framework "
                          "or does not have at least one assessed domain"), 400))
 
-    # Check if brief response already exists from this supplier
-    if BriefResponse.query.filter(BriefResponse.supplier == supplier, BriefResponse.brief == brief).first():
-        abort(make_response(jsonify(
-            errorMessage="Brief response already exists for supplier '{}'".format(supplier.code)), 400))
+    lot = lots_service.first(slug='digital-professionals')
+    if brief.lot_id == lot.id:
+        # Check if there are more than 3 brief response already from this supplier when professional aka specialists
+        brief_response_count = brief_responses_service.find(supplier_code=supplier.code,
+                                                            brief_id=brief.id,
+                                                            withdrawn_at=None).count()
+        if (brief_response_count > 2):  # TODO magic number
+            abort(make_response(jsonify(
```

Comment:
if have done a json `abort` somewhere else - use that

Reply:
fixed in latest commit

Reply:
used abort from helper. created forbidden and not found also